src/components/feature_activation/Features.js

```diff
@@ -231,6 +234,131 @@ class Features extends React.Component {
       </div>
     );
   }
+
+  hasBefore = () => this.state.page > 1;
+
+  hasAfter = () => this.state.page < this.state.pages.length;
+
+  renderNewUi() {
+    const loadPagination = () => {
+      if (this.state.pages.length === 0) {
+        return null;
+      }
+      return (
+        <div className="tx-pagination-btn no-padding">
+          <Link
+            className={!this.hasBefore() ? 'page-link-btn-disable' : 'page-link-btn-enable'}
+            to={this.pagination.setURLParameters({ page: this.state.page - 1 })}
+          >
+            <button
+              className={
+                !this.hasBefore() ? 'tx-previous-btn disable-button' : 'tx-next-btn tx-disabled'
+              }
+              disabled={!this.hasBefore()}
+            >
+              Previous
+            </button>
+          </Link>
+          <Link
+            className={!this.hasAfter() ? 'page-link-btn-disable' : 'page-link-btn-enable'}
+            to={this.pagination.setURLParameters({ page: this.state.page + 1 })}
+          >
+            <button
+              className={
+                !this.hasAfter() ? 'tx-previous-btn disable-button' : 'tx-next-btn tx-enable'
+              }
+              disabled={!this.hasAfter()}
+            >
+              Next
+            </button>
+          </Link>
+        </div>
+      );
+    };
+
+    const loadTable = () => {
+      return (
+        <div className="table-responsive mt-5">
+          <table className=" table-stylized table-features" id="features-table">
+            <thead>
+              <tr>
+                <th className="d-lg-table-cell">Name</th>
+                <th className="d-lg-table-cell">State</th>
+                <th className="d-lg-table-cell td-mobile">Acceptance</th>
+                <th className="d-lg-table-cell td-mobile">Threshold</th>
+                <th className="d-lg-table-cell td-mobile">Start Height</th>
+                <th className="d-lg-table-cell td-mobile">Minimum Activation Height</th>
+                <th className="d-lg-table-cell td-mobile">Timeout Height</th>
+                <th className="d-lg-table-cell td-mobile">Lock-in on Timeout</th>
+                <th className="d-lg-table-cell td-mobile">Since Version</th>
+                <th className="d-lg-table-cell arrow-td-mobile"></th>
+              </tr>
+            </thead>
```

Comment:
Adjusted table responsiveness to ensure proper display near mobile view without breaking layout. Please review!

Reply:
Since we're using the lateral scroll solution, all header titles should not break anymore, as their minimum width is always guaranteed. However, some of them are still breaking the words:

![image](https://github.com/user-attachments/assets/b7e09a34-5a56-42f5-bf1b-c04139f4e956)

Reply:
Fixed!